experimental/PropertyDDS/examples/schemas/tsconfig.json

```diff
@@ -1,9 +1,8 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../common/build/build-common/tsconfig.node16.json",
 	"compilerOptions": {
-		"composite": true,
 		"rootDir": "./src",
-		"outDir": "./dist",
+		"outDir": "./lib",
 		"baseUrl": ".",
```

Comment:
What is baseUrl doing?

Reply:
schemas src code and tests already use relative paths, so it should be fine to remove this line. I will remove the line and test.